src/Host/Client/Impl/Definitions/IRSessionCallback.cs

```diff
@@ -35,7 +36,18 @@ public interface IRSessionCallback {
         /// Set locator mode in the plot window
```

Comment:
Maybe we should start splitting `IRSessionCallback` and have separate `IRSessionPlotCallback`? @int19h what do you think?